wiki/markdown/configuration.md

```diff
@@ -0,0 +1,61 @@
+# Configuration
+
+To be able to execute and use Eiffel-Intelligence Front-end, some configuration need to provided.
```

Comment:
Here front-end is hyphened but with first capital letter, later no hyphen is used etc.. Please change all occurrences to use "front-end" to be consistent with other documentation.

Reply:
fixed.